subsys/spm/Kconfig

```diff
@@ -111,6 +111,13 @@ config SPM_SERVICE_PREVALIDATE
 	  secure service is needed for the app to access the prevalidation
 	  function.
 
+config SPM_SERVICE_BUSY_WAIT
+	bool "Busy wait in secure mode (debug function)"
+	default y
+	help
```

Comment:
Could it depend on ZTEST so that its not visible for customers when running menuconfig on a normal project?

Reply:
Yes this is an option. (or TEST)
How about samples, where we would like to have this set by default as well?

Reply:
Sadly, this config (in SPM) cannot depend on a config in the app.

Reply:
But what if we modify the build system to pass "CONFIG_ZTEST" to any child image? In my opinion that is a reasonable feature. If you agree we can create a task and add it to the backlog.

Reply:
I spent a bit of time thinking about this yesterday. I had a similar thought. We have the option of passing `-Dspm_CONFIG_TEST=y`-style Kconfig overrides from app to spm, but it's quite blunt, and means that those configs will be ignored if set with e.g. `ninja spm_menuconfig`.

I also considered just doing it with the SPM_SERVICE configs since they anyway have particular requirements about being in sync across app and spm. Doing this would solve another problem as well, which I fix in https://github.com/nrfconnect/sdk-nrf/pull/2710, but I like this solution better.

Reply:
The SPM_SERVICE_* configs are now passed from app to spm